### PR TITLE
fix: sentence casing fixes

### PR DESCRIPTION
**Pull request by jpvajda**
This PR makes numerous casing fixes and resolved misc typos in our install plans.

### CODE REVIEW

File: install/aws/aws-cloudwatch-metric-streams/install.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 id: aws-cloudwatch-metric-streams
-name: AWS-CloudWatch-Metric-Streams
-title: AWS Metric Streams
+name: AWS CloudWatch Metric Streams
+title: AWS CloudWatch Metric Streams
```
Comment (josephgregoryii) [inline install/aws/aws-cloudwatch-metric-streams/install.yml:3]: Double checked this on newrelic.io - Knowing the pre-fix issues and with the upcoming patch for it, i _believe_ this shouldn't cause any issues

**Review by josephgregoryii — APPROVED** [review]
Looks good to me

**Comment by jpvajda** [comment]
I've asked DevEx to confirm that these changes won't impact the functionality of install plans, just to double check.

**Review by tyreer — APPROVED** [review]
LGTM 

Correct, the service only uses `id` for the slug: https://source.datanerd.us/nr1-dev-experience/catalog-service-elixir/blob/master/apps/api/lib/api/schema.ex#L644-L645

**Pull request merged**